static/js/impala/persona_creation.js

```diff
@@ -217,21 +217,21 @@
                 });
             };
 
-        $d.delegate('.reset', 'click', _pd(function() {
+        $d.on('click', '.reset', _pd(function() {
             var $this = $(this),
                 $p = $this.closest('.row');
             $p.find('input[type="hidden"]').val('');
             $p.find('input[type=file], .note').show();
-            $p.find('.preview').removeAttr('src').removeClass('loaded');
```

Comment:
Historically setting an image src attr to an empty string used to make a request to the parent document. Playing with that in jsbin if you check the src after setting it to '' it looks to be set to the url of the parent.